app/src/main/java/com/example/arc_exapmle/note/NoteRepository.kt

```diff
@@ -5,132 +5,100 @@ import android.os.AsyncTask
 import androidx.lifecycle.LiveData
 import com.example.arc_exapmle.StarDatabase
 import com.example.arc_exapmle.user.UserEntity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.Default
+import kotlinx.coroutines.launch
 
-class NoteRepository(application: Application , val userId : Int) {
-    private var allNotes: LiveData<List<NoteEntity>>
+class NoteRepository(application: Application, val userId: Int) {
```

Comment:
gharar bod be jaaye application khode database ya DAO ro pass bedim ha!

Reply:
Hal shod